src/app/facebook-pixel-service/facebook-pixel-service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FacebookPixelService {
+  private loadOk = false;
+
+  constructor() {}
+
+  load() {
+    if (!this.loadOk) {
+      (function(f: any, b, e, v, n, t, s) {
```

Comment:
@NoelLH Any ideas how I can resolve this warning `on-arrow functions are forbidden` think this is causing the rest of the code to not compile correctly, for example line 17 is throwing `Type '() => void' is not assignable to type 'undefined'`

Reply:
Okay think i've managed to convert this to an arrow function